src/Standards/ISAAC/Sniffs/Classes/MethodPerClassLimitSniff.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace IsaacCodingStandard\Standards\ISAAC\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use SlevomatCodingStandard\Helpers\FunctionHelper;
+use SlevomatCodingStandard\Helpers\SniffSettingsHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+use function count;
+use function sprintf;
+
+use const T_ANON_CLASS;
+use const T_CLASS;
+use const T_FUNCTION;
+use const T_INTERFACE;
+use const T_TRAIT;
+
+class MethodPerClassLimitSniff
+{
+    public const CODE_METHOD_PER_CLASS_LIMIT = 'MethodPerClassLimit';
+
+    public const ERROR_MESSAGE = '%s has too many methods: %d. Can be up to %d methods.';
+
+    /** @var int */
+    public $maxMethodCount = 10;
+
+    /**
+     * @return array<int, (int|string)>
+     */
+    public function register(): array
+    {
+        return [T_CLASS, T_ANON_CLASS, T_TRAIT, T_INTERFACE];
+    }
+
+    /**
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
```

Comment:
I think this can be removed. I removed this line and could ran `vendor/bin/phpcs` without errors.

Reply:
Removed.

However, when running phpcs using PHP 7.4 or 8.0 a `MissingNativeTypeHint` violation was reported for another line, so I added that.